10-JavaScript/xx-objects.md

```diff
@@ -11,7 +11,7 @@ We spoke briefly about objects in our introductory conversation, but there's way
 So far, every object we've built has been defined explicitly in our code. This is very similar to how hashes work in Ruby. Objects can contain any sort of variable, including arrays, functions and even other objects. Remember that inside an object, you can use `this` to refer to that object, similar to Ruby's `self`. Here's an example, just to review:
```

Comment:
I think this is an incorrect rename - it should be just objects.md not xx-objects.md